tidb-cloud/tidb-cloud-quickstart.md

```diff
@@ -35,100 +29,65 @@ To create a free Developer Tier cluster, take the following steps:
-    The cluster creation process starts and the **Security Settings** dialog box is displayed.
+4. Add the table filter rules if needed. For the sample data, you can skip this step. Click **Next**.
 
-10. In the **Security Settings** dialog box, set the root password and allowed IP addresses to connect to your cluster, and then click **Apply**.
+5. On the **Preview** page, confirm the data to be imported and then click **Start Import**.
 
-    Your TiDB Cloud cluster will be created in approximately 5 to 15 minutes.
+The data import process will take several minutes. When the data import progress shows **Finished**, you have successfully imported the sample data and the database schema to your database in TiDB Cloud.
 
-</div>
-</SimpleTab>
+## Step 4. Connect to your TiDB cluster
 
-## Step 2. Connect to your TiDB cluster
+After loading data to the cluster, you can connect to your cluster from the command line or with a programming language.
 
 1. Navigate to the **Clusters** page.
 
 2. In the area of your newly created cluster, click **Connect** in the upper-right corner. A connection dialog is displayed.
 
-    > **Tip:**
-    >
-    > Alternatively, you can also click the name of your newly created cluster on the **Clusters** page, and then click **Setup** in the **Connection** area.
-
 3. Follow the instructions in the dialog to connect to your TiDB cluster.
 
-    <SimpleTab>
-    <div label="Developer Tier">
+    1. Create traffic filter for your connection.
 
-    To connect to a Developer Tier cluster, check **Step 2: Connect with a SQL client** in the dialog, click the tab of your preferred connection method, and then connect to your cluster with the connection string.
+    2. Use a SQL client to connect to your cluster. Click the tab of your preferred connection method, and then connect to your cluster with the connection string.
 
-    > **Note:**
-    >
-    > - For [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier), when you connect to your cluster, you must include the prefix for your cluster in the user name and wrap the name with quotation marks. For more information, see [User name prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix).
-    > - TiDB Cloud is MySQL-compatible, so you can connect to your cluster using any MySQL client tools. We recommend using [mysql — The MySQL Command-Line Client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html) or [mysql — The MySQL Command-Line Client from MariaDB](https://mariadb.com/kb/en/mysql-command-line-client/).
-
-    </div>
-
-    <div label="Dedicated Tier">
-
-    To connect to a Dedicated Tier cluster, take the following steps:
+    For [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier), when you connect to your cluster, you must include the prefix for your cluster in the user name and wrap the name with quotation marks. For more information, see [User name prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix).
```

Comment:
```suggestion

```

Reply:
This sentence can be removed now because the same information has been added to the UI.